system_metrics_collector/test/system_metrics_collector/test_constants.hpp

```diff
@@ -20,15 +20,24 @@
  */
 namespace test_constants
 {
-constexpr const double MEMORY_USED_PERCENTAGE = 44.148416198995363;
+constexpr const std::chrono::milliseconds TEST_LENGTH =
+  std::chrono::milliseconds(250);
+constexpr const std::chrono::milliseconds MEASURE_PERIOD =
+  std::chrono::milliseconds(50);
+constexpr const std::chrono::milliseconds PUBLISH_PERIOD =
+  std::chrono::milliseconds(80);
```

Comment:
Nit: can be less verbose as
```c++
constexpr const std::chrono::milliseconds TEST_LENGTH{250};
```

Reply:
```c++
using std::chrono_literals::ms;
constexpr auto TEST_LENGTH = 250ms;
```

;)

Reply:
Is this kind of `using` statement okay to use in a header file?

Reply:
Inside the namespace it's fine because you're controlling it end-to-end. But not at the root level outside of a namespace where you don't know actually what you can collide with.

Reply:
I think I'm going to hold off on using the `std::chrono_literals` for now, because GCC emits a warning on the `using std::chrono_literals::operator""ms` statement ([prior to GCC 8](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65923)).